example/config/README.md

```diff
@@ -0,0 +1,249 @@
+    - `test_label_timespans`: time period across which outcomes are labeled in test matrices
+
+### Cohort Config
+Cohorts are configured by passing a query with placeholders for the **as_of_date**.
+
+- `cohort_conifg`: 
+    - `qurey`: The `query` key should have a query, parameterized with an `'{as_of_date}'`, to select the entity_ids that should be included for a given date. The `{as_of_date}` will be replaced with each `as_of_date` that the experiment needs. The returned `entity_id` must be an integer.
+    - `name`: You may enter a `name` for your configuration. This will be included in the metadata for each matrix and used to group models. If you don't pass one, the string `default` will be used.
+
+
+### Label Generation
+Labels are configured by passing a query with placeholders for the `as_of_date` and `label_timespan`.
+
+- `label_config`:
+    - `query`: The query must return two columns: `entity_id` and `outcome`, based on a given `as_of_date` and `label_timespan`. The `as_of_date` and `label_timespan` must be represented by placeholders marked by curly brackets. The example below reproduces the inspection outcome boolean-or logic: In addition, you can configure what label is given to entities that are in the matrix (see **cohort_config** section) but that do not show up in this label query. By default, these will show up as missing/null.
+    - `include_missing_labels_in_train_as`: However, passing the key `include_missing_labels_in_train_as` allows you to pick True or False.
+    - `name`: In addition to these configuration options, you can pass a name to apply to the label configuration that will be present in matrix metadata for each matrix created by this experiment, under the `label_name` key. The default label_name is `outcome`.
+
+
+### Feature Generation
+The aggregate features to generate for each train/test split. Implemented by wrapping [collate](https://github.com/dssg/triage/tree/config_doc/src/triage/component/collate). Most terminology here is taken directly from [collate](https://github.com/dssg/triage/tree/config_doc/src/triage/component/collate).
+
+Each entry describes a collate.SpacetimeAggregation object, and the arguments needed to create it. Generally, each of these entries controls the features from one source table, though in the case of multiple groups may result in multiple output tables.
+
+Rules specifying how to handle imputation of null values must be explicitly defined in your config file. These can be specified in two places: either within each feature or overall for each type of feature (aggregates_imputation, categoricals_imputation, array_categoricals_imputation). In either case, a rule must be given for each aggregation function (e.g., sum, max, avg, etc) used, or a catch-all can be specified with `all`. Aggregation function-specific rules will take precedence over the `all` rule and feature-specific rules will take precedence over the higher-level rules. Several examples are provided below.
+
```

Comment:
It might make sense to add list of available/supported aggregation functions/metrics (definitely note that the terms are synonymous since we use both). That could just be a note that the supported ones are whatever aggregation functions are in your version of postgres, plus any user defined functions.